### PR TITLE
TSSO: Technical Security Standard Ontology

### PR DESCRIPTION
**Pull request by JoseIllSCCH**
The resulting ontology serves as the basis to represent technical standards in a machine readable way. It provides elements to define publications, their provisions, their dependencies and possible constraints, among other constructs. This ontology is based on three technical standars, namely the IEC 62443 4-2, ETSI 303645 and the more general NIST Framework for Improving Critical Infrastructure Cybersecurity.

### CODE REVIEW

**Comment by JoseIllSCCH** [comment]
> @JoseIllSCCH please add your github id so we know it's you next time. Thanks!

Hello. Where should i add my github id? to the readme, commit message, both, or somewhere else? Thanks in advance.

**Comment by dgarijo** [comment]
Readme, as you have done now :)

**Comment by dgarijo** [comment]
I am merging, but I think line 7 has an error in the rewrite rule.

File: tsso/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* Gandalf Denk, [gandalf.denk@limessecurity.at](mailto:gandalf.denk@limessecurity.at)
+
+## Info
+
+* The homepage including the documentation will be soon be available. At the moment only a local version is provided at the github repo.
+
+## Mantainer
+
+* Jose Illescas [jose.illescas@scch.at]
+* Github [JoseIllSCCH](https://github.com/JoseIllSCCH)
```
Comment (TallTed) [inline tsso/README.md:25]: 1. fix typo
2. fix bullet nesting
```suggestion
## Maintainer

* Jose Illescas [jose.illescas@scch.at]
  * Github [JoseIllSCCH](https://github.com/JoseIllSCCH)
```

**Pull request merged**